src/window.rs

```diff
@@ -287,7 +287,7 @@ impl WindowBuilder {
         self.window.inner_size = Some(self.window.inner_size.unwrap_or_else(|| {
             if let Some(ref monitor) = self.window.fullscreen {
                 // resizing the window to the dimensions of the monitor when fullscreen
-                LogicalSize::from_physical(monitor.size(), 1.0)
+                LogicalSize::from_physical(monitor.dimensions(), monitor.hidpi_factor()) // DPI factor applies here since this is a borderless window and not real fullscreen
```

Comment:
@Osspial I think this ought to be `.size()` now! I can fix this, unless you're already handling it?

Reply:
You're right! Feel free to go ahead and commit that change.

Reply:
Done!